README.md

```diff
@@ -364,6 +364,18 @@ with double quotes and sometimes it is necessary.
 
 - For `initscripts`,  `mac` is the currently configured MAC address of the device (`HWADDR`).
 
+### `cloned_mac`
+
+The `cloned_mac` address is optional and allow to specify the strategy to get the default
+mac or to set your own mac. The value of the `cloned_mac` address needs to be specified in
```

Comment:
How about "Besides explicitly specifying the value as a MAC address with hexadecimal notation, the following special values are also supported:" ?

Reply:
done